frame/evm/src/lib.rs

```diff
@@ -148,6 +148,12 @@ pub trait IssuingHandler {
 	fn handle(address: H160, caller: H160, input: &[u8]) -> DispatchResult;
 }
 
+impl IssuingHandler for () {
+	fn handle(_: H160, _: H160, _: &[u8]) -> DispatchResult {
+		Ok(())
```

Comment:
Is this expected?

Reply:
`type IssuingHandler = ();` only be used to replace `EmptyIssuingHandler`, just for the test scenario.

Reply:
Please add some docs.